client/db/bolt/db_test.go

```diff
@@ -125,7 +125,7 @@ func TestStorePrimaryCredentials(t *testing.T) {
 		t.Fatalf("no error for missing credentials: %v", err)
 	}
 
-	newCreds := func(seed, inner, innerParams, outerParams bool) *db.PrimaryCredentials {
+	newCreds := func(seed, inner, innerParams, outerParams, encRecoverySeed, encRecoverySeedP bool) *db.PrimaryCredentials {
```

Comment:
New arguments appear to do nothing.

Reply:
Thanks! These are residual changes from previous changes.